bear/main.py.in

```diff
@@ -589,7 +625,10 @@ class Compilation:
 
         if command:  # not empty list will allow to index '0' and '1:'
             executable = os.path.basename(command[0])  # type: str
```

Comment:
I think this change is not intended. This was hint the type checker about the `parameters` is a list of strings. If you move this into another line, it won't mean the same thing.

Reply:
It really has to be like this.
```
parameters = command[1:]  # type: List[str]
```
Could you please revert this line change?